setup.cfg

```diff
@@ -0,0 +1,3 @@
+[bdist_wheel]
```

Comment:
It's for "wheel" built-package format for Python. 
These will make wheel build a universal wheel file (e.g. django-social-hashtags-py2.py3-none-any.whl) and you won’t have to circle through virtual environments of all supported Python versions to build them separately.